### PR TITLE
[8.0] Watchdog fix: always report correct types

### LINKED ISSUE #6935: Exception raised when inserting an unknown `LastUpdateCPU(s)` fied in `ElasticJobParametersDB`
I randomly found this exception yesterday:

```bash
2023-03-27 09:48:07 UTC WorkloadManagement/JobStateUpdate/WorkloadManagement/ElasticJobParametersDB ERROR: Couldn't insert or update data RequestError(400, 'mapper_parsing_exception', "failed to parse field [LastUpdateCPU(s)] of type [float] in document with id '123456789'. Preview of field's value: 'Could not be estimated'")
2023-03-27 09:48:07 UTC WorkloadManagement/JobStateUpdate ERROR: Failed to add Job Parameters to ElasticJobParametersDB RequestError(400, 'mapper_parsing_exception', "failed to parse field [LastUpdateCPU(s)] of type [float] in document with id '123456789'. Preview of field's value: 'Could not be estimated'")
2023-03-27 09:48:07 UTC WorkloadManagement/JobStateUpdate NOTICE: Returning response [user:johndoe] (0.03 secs) ERROR: RequestError(400, 'mapper_parsing_exception', "failed to parse field [LastUpdateCPU(s)] of type [float] in document with id '123456789'. Preview of field's value: 'Could not be estimated'")

  File "/opt/dirac/versions/v11.0.7/Linux-x86_64/lib/python3.9/threading.py", line 937, in _bootstrap
    self._bootstrap_inner()
  File "/opt/dirac/versions/v11.0.7/Linux-x86_64/lib/python3.9/threading.py", line 980, in _bootstrap_inner
    self.run()
  File "/opt/dirac/versions/v11.0.7/Linux-x86_64/lib/python3.9/threading.py", line 917, in run
    self._target(*self._args, **self._kwargs)
  File "/opt/dirac/versions/v11.0.7/Linux-x86_64/lib/python3.9/concurrent/futures/thread.py", line 83, in _worker
    work_item.run()
  File "/opt/dirac/versions/v11.0.7/Linux-x86_64/lib/python3.9/concurrent/futures/thread.py", line 58, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/opt/dirac/versions/v11.0.7/Linux-x86_64/lib/python3.9/site-packages/DIRAC/Core/DISET/private/Service.py", line 349, in _processInThread
    result = self._processProposal(trid, proposalTuple, handlerObj)
  File "/opt/dirac/versions/v11.0.7/Linux-x86_64/lib/python3.9/site-packages/DIRAC/Core/DISET/private/Service.py", line 535, in _processProposal
    result = self._executeAction(trid, proposalTuple, handlerObj)
  File "/opt/dirac/versions/v11.0.7/Linux-x86_64/lib/python3.9/site-packages/DIRAC/Core/DISET/private/Service.py", line 555, in _executeAction
    response = handlerObj._rh_executeAction(proposalTuple)
  File "/opt/dirac/versions/v11.0.7/Linux-x86_64/lib/python3.9/site-packages/DIRAC/Core/DISET/RequestHandler.py", line 120, in _rh_executeAction
    retVal = self.__doRPC(actionTuple[1])
  File "/opt/dirac/versions/v11.0.7/Linux-x86_64/lib/python3.9/site-packages/DIRAC/Core/DISET/RequestHandler.py", line 251, in __doRPC
    return self.__RPCCallFunction(method, args)
  File "/opt/dirac/versions/v11.0.7/Linux-x86_64/lib/python3.9/site-packages/DIRAC/Core/DISET/RequestHandler.py", line 292, in __RPCCallFunction
    uReturnValue = oMethod(*args)
  File "/opt/dirac/versions/v11.0.7/Linux-x86_64/lib/python3.9/site-packages/DIRAC/Core/Utilities/DEncode.py", line 90, in inner
    return meth(*args, **kwargs)
  File "/opt/dirac/versions/v11.0.7/Linux-x86_64/lib/python3.9/site-packages/DIRAC/WorkloadManagementSystem/Service/JobStateUpdateHandler.py", line 393, in export_setJobParameters
    result = cls.elasticJobParametersDB.setJobParameters(int(jobID), parameters)
  File "/opt/dirac/versions/v11.0.7/Linux-x86_64/lib/python3.9/site-packages/DIRAC/WorkloadManagementSystem/DB/ElasticJobParametersDB.py", line 205, in setJobParameters
    result = self.updateDoc(index=self._indexName(jobID), docID=str(jobID), body={"doc": parametersDict})
  File "/opt/dirac/versions/v11.0.7/Linux-x86_64/lib/python3.9/site-packages/DIRAC/Core/Utilities/ElasticSearchDB.py", line 46, in wrapper_decorator
    return method(self, *args, **kwargs)
  File "/opt/dirac/versions/v11.0.7/Linux-x86_64/lib/python3.9/site-packages/DIRAC/Core/Utilities/ElasticSearchDB.py", line 277, in updateDoc
    return S_ERROR(re)
```

The `LastUpdateCPU(s)` is set by the `Watchdog`: https://github.com/DIRACGrid/DIRAC/blob/integration/src/DIRAC/WorkloadManagementSystem/JobWrapper/Watchdog.py#L841

Should this be modified at this level?

### PR DESCRIPTION
**Pull request by fstagni**
closes #6935 

BEGINRELEASENOTES

*WMS
FIX: Watchdog fix: always report correct types

ENDRELEASENOTES

### CODE REVIEW

File: src/DIRAC/WorkloadManagementSystem/JobWrapper/Watchdog.py [hunk 1]
```diff
@@ -834,31 +834,28 @@ def __getUsageSummary(self):
...
             space = self.parameters["DiskSpace"]
             if space:
-                value = abs(float(space[-1]) - float(self.initialValues["DiskSpace"]))
-                if value < 0:
-                    value = 0
-                summary["DiskSpace(MB)"] = value
+                summary["DiskSpace(MB)"] = max(abs(float(space[-1]) - float(self.initialValues["DiskSpace"])), 0.0)
             else:
-                summary["DiskSpace(MB)"] = "Could not be estimated"
+                summary["DiskSpace(MB)"] = 0.0
```
Comment (chrisburr) [inline src/DIRAC/WorkloadManagementSystem/JobWrapper/Watchdog.py]: Would `math.nan` be a better fallback value?

Comment (fstagni) [reply src/DIRAC/WorkloadManagementSystem/JobWrapper/Watchdog.py]: I really have no idea which type `math.nan` would become on the OpenSearch server - the mapping might end up being incorrect anyway. I don't see big issues into using a safer "0.0" here.

Comment (chrisburr) [reply src/DIRAC/WorkloadManagementSystem/JobWrapper/Watchdog.py]: It's uses IEEE 754 floats so it's just as valid as `0.0`: https://opensearch.org/docs/latest/field-types/numeric/

I already checked that DEncode works correctly with `math.nan`.

Comment (fstagni) [reply src/DIRAC/WorkloadManagementSystem/JobWrapper/Watchdog.py]: OK, updated with it.

**Review by chrisburr — APPROVED** [review]

**Review by Lorenzov1999 — APPROVED** [review]

**Pull request merged**